src/pipx/commands/run.py

```diff
@@ -146,12 +151,29 @@ def _download_and_run(
 
     if not venv.has_app(app, app_filename):
         apps = venv.pipx_metadata.main_package.apps
-        raise PipxError(
-            f"""
-            '{app}' executable script not found in package '{package_or_url}'.
-            Available executable scripts: {', '.join(b for b in apps)}
-            """
-        )
+
+        # If there's a single app inside the package, run that by default
+        if len(apps) == 1:
+            app = apps[0]
+            print(f"NOTE: running app {app!r} from {package_name!r}")
+            if WINDOWS:
+                app_filename = f"{app}.exe"
+                logger.info(f"Assuming app is {app_filename!r} (Windows only)")
+            else:
+                app_filename = app
```

Comment:
This probably should not use `print`. Also, the Windows only line can just be combined to the previous log (something like *running app 'foo' (assuming 'foo.exe' on Windows) from 'foo-package'*).

Reply:
A slight concern about this: Shouldn't the user be notified that an app whose name is different from what he just typed is being run?
Changing the `print` to `logging.info` will stop it from showing up to the user. Is that intended?

Reply:
Sorry, I was confused between different logger setups between pip and pipx (by default `INFO` is shown in pip, but `pipx` only shows `WARNING`). Using `print` is correct here.